recipe/meta.yaml

```diff
@@ -1,44 +1,32 @@
 {% set name = "pigz" %}
 {% set version = "2.3.4" %}
-{% set sha256 = "6f031fa40bc15b1d80d502ff91f83ba14f4b079e886bfb83221374f7bf5c8f9a" %}
+{% set sha256 = "763f2fdb203aa0b7b640e63385e38e5dd4e5aaa041bc8e42aa96f2ef156b06e8" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
   fn: {{ name }}-{{ version }}.tar.gz
-  url: https://zlib.net/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  url: https://github.com/madler/{{ name }}/archive/v{{ version }}.tar.gz
```

Comment:
This version is no longer available from https://zlib.net so have switched to the maintainer's GitHub repo. Would upgrade to the next version however that uses `fsync`, which does not work on Windows. Please see issue ( https://github.com/madler/pigz/issues/54 ) for details. Given there are already a lot of changes and it is probably worthwhile for the maintainers of this feedstock to approach that issue separately, have stuck to the current version so that we can get this feedstock up-to-date at least.